website/docs/d/nsxt_dynamic_security_group.html.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_dynamic_security_group"
+sidebar_current: "docs-vcd-data-source-nsxt-dynamic-security-group"
+description: |-
+  Provides a data source to read NSX-T Dynamic Security Groups. Dynamic Security Groups group virtual
+  machines based on specific criteria (VM Names or Security tags) to which distributed firewall rules
```

Comment:
For easier eye scanning:

```suggestion
  Provides a data source to read NSX-T Dynamic Security Groups. Dynamic Security Groups group Virtual
  Machines based on specific criteria (VM Names or Security tags) to which Distributed Firewall Rules
```

Reply:
Adjusted. Also spotted incorrect grammar and fixed.